tide/wall/SVGTiler.cpp

```diff
@@ -39,20 +39,38 @@
 
 #include "SVGTiler.h"
 
+#include "VectorialContent.h"
+
+#include <QThread>
+
 namespace
 {
-const uint tileSize = 512;
+const uint tileSize = 1024;
 }
 
-SVGTiler::SVGTiler( SVGTextureFactoryPtr factory )
-    : LodTiler( factory->getMaxSize(), tileSize )
-    , _factory( std::move( factory ))
+SVGTiler::SVGTiler( SVG& svg )
+    : LodTiler( svg.getSize() * VectorialContent::getMaxScale(), tileSize )
+    , _svg( svg )
 {}
 
 QImage SVGTiler::getCachableTileImage( const uint tileId ) const
 {
     const QRect imageRect = getTileRect( tileId );
     const QRectF zoomRect = getNormalizedTileRect( tileId );
 
-    return _factory->createTexture( imageRect.size(), zoomRect );
+#if TIDE_USE_CAIRO && TIDE_USE_RSVG
```

Comment:
this cannot be abstracted via the backend? It's a bit sad to 'leak' this here :)